src/rulesets/jsonapi.ts

```diff
@@ -249,7 +249,7 @@ export const rules = {
     );
   },
   compoundDocuments: ({ responses }: SnykApiCheckDsl) => {
-    responses.requirement.must(
+    responses.requirementOnChange.must(
```

Comment:
as written, any changes to they make to the response body, would require people to add self links for CI to pass.

Is that what you want?

If you only want new responses to follow this use added